### DISCUSSION TITLE
小明的OJ 论坛 喜报

### DISCUSSION THREAD

**Discussion by PythonSmall-Q** [langningchen/XMOJ-Script#129] [General]
https://www.bing.com/search?q=%e5%b0%8f%e6%98%8e%e7%9a%84OJ+%e8%ae%ba%e5%9d%9b&PC=U316&FPIG=7A6CC2EBF3B54E058FB25AC867585202&first=1&FORM=PERE
wc我能在bing搜到我的论坛
在google也行
https://www.google.com/search?q=%E5%B0%8F%E6%98%8E%E7%9A%84OJ+%E8%AE%BA%E5%9D%9B&oq=%E5%B0%8F%E6%98%8E%E7%9A%84OJ+%E8%AE%BA%E5%9D%9B&gs_lcrp=EgZjaHJvbWUyBggAEEUYQDIGCAEQRRg50gEJMTEwMTlqMGoxqAIAsAIA&sourceid=chrome&ie=UTF-8

**Comment by PythonSmall-Q** [comment]
@langningchen @boomzero

**Comment by PythonSmall-Q** [comment]
题外话不会删吧

**Comment by langningchen** [comment]
不删不删

**Comment by boomzero** [comment]
> https://www.bing.com/search?q=%e5%b0%8f%e6%98%8e%e7%9a%84OJ+%e8%ae%ba%e5%9d%9b&PC=U316&FPIG=7A6CC2EBF3B54E058FB25AC867585202&first=1&FORM=PERE wc我能在bing搜到我的论坛 在google也行 https://www.google.com/search?q=%E5%B0%8F%E6%98%8E%E7%9A%84OJ+%E8%AE%BA%E5%9D%9B&oq=%E5%B0%8F%E6%98%8E%E7%9A%84OJ+%E8%AE%BA%E5%9D%9B&gs_lcrp=EgZjaHJvbWUyBggAEEUYQDIGCAEQRRg50gEJMTEwMTlqMGoxqAIAsAIA&sourceid=chrome&ie=UTF-8

SEO 不错
